packages/amplify_datastore/example/android/app/build.gradle

```diff
@@ -23,11 +23,13 @@ if (flutterVersionName == null) {
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-parcelize'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     // TODO(dnys1): Replace with `flutter.compileSdkVersion` when bumping min Flutter version
-    compileSdkVersion 33
+    namespace 'com.amazonaws.amplify.amplify_datastore_example'
+    compileSdkVersion 34
```

Comment:
Q: does this key change in a newer Flutter version? Does it need to be updated?

Reply:
I get a `Caused by: java.lang.RuntimeException: Namespace not specified. Specify a namespace in the module's build file.` error if I don't add in the namespace

Reply:
I was referring to `compileSdkVersion` being renamed to `compileSdk`. Couldn't find this in the change log, but creating a new flutter project uses that key. Curious if this can be updated now?